dvc/exceptions.py

```diff
@@ -1,3 +1,2 @@
 class DvcException(Exception):
-    def __init__(self, msg):
```

Comment:
Can you pass a message without this constructor?

Reply:
Yes, sure. Class 'Exception' has the same __init__ function, so there is no need to override it.